src/main/kotlin/com/lowbudgetlcs/api/routes/v1/player/PlayerEndpointsV1.kt

```diff
@@ -0,0 +1,80 @@
+package com.lowbudgetlcs.api.routes.v1.player
+
+import com.lowbudgetlcs.api.dto.players.*
+import com.lowbudgetlcs.api.setCidContext
+import com.lowbudgetlcs.domain.models.riot.account.toRiotAccountId
+import com.lowbudgetlcs.domain.models.player.toPlayerId
+import com.lowbudgetlcs.domain.services.player.PlayerService
+import io.ktor.http.*
+import io.ktor.server.application.Application
+import io.ktor.server.request.*
+import io.ktor.server.resources.*
+import io.ktor.server.resources.patch
+import io.ktor.server.resources.post
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+private val logger: Logger = LoggerFactory.getLogger(Application::class.java)

```

Comment:
WHAT IS WITH THE `APPLICATION::CLASS.JAVA` WHAT IS HAPPENING?
AND WHY IS GITHUB SAYING THIS IS A NEW CLASS?